### PR TITLE
21134 Added continuation in types for restoration + updated Iran and Turkey

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:*
- bcgov/entity#21134
- bcgov/entity#21568
- bcgov/entity#21482
- and Ops ticket 3885

*Description of changes:*
- app version = 5.5.4
- updated description for Iran
- updated description for Turkey
- added C, CBEN, CCC, and CUL to list of restorable businesses
- lint fixes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/list-data/intl-jurisdictions.ts
Comment (severinbeauvais) [inline src/list-data/intl-jurisdictions.ts:1]: These are fixes for bcgov/entity#21568.

**Review by eve-git — APPROVED** [review]
LGTM

File: src/list-data/request-action-mapping.ts [hunk 1]
```diff
@@ -1,23 +1,6 @@
...
-  EntityTypes.LP,
-  EntityTypes.LL,
-  EntityTypes.CP,
-  EntityTypes.BC,
-  EntityTypes.CC,
-  EntityTypes.SO,
-  EntityTypes.PA,
-  EntityTypes.FI,
-  EntityTypes.PAR
-]
```
Comment (severinbeauvais) [inline src/list-data/request-action-mapping.ts:19]: This was only used once so I moved it to lines 41-60.

File: src/list-data/request-action-mapping.ts [hunk 2]
```diff
@@ -42,17 +25,39 @@ export const BcMapping: RequestActionMappingI = {
...
+    EntityTypes.CC,
+    EntityTypes.SO,
+    EntityTypes.PA,
+    EntityTypes.FI,
+    EntityTypes.PAR,
+    EntityTypes.C,
+    EntityTypes.CBEN,
+    EntityTypes.CCC,
+    EntityTypes.CUL
+  ],
```
Comment (severinbeauvais) [inline src/list-data/request-action-mapping.ts:60]: I'm pretty sure only the REH and CHG arrays are used from here, so those are the only ones I updated.

**Comment by severinbeauvais** [comment]
I also added C/CBEN/CCC/CUL to various feature flags.

# Supported Restoration Entities

image

# Supported Name Change Entities

image

# Supported Amalgamation Entities

image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-768-hau52psz.web.app

**Comment by severinbeauvais** [comment]
Note to self: compare Dev vs local for creating a restoration NR for C0848313.

**Comment by severinbeauvais** [comment]
In Dev:

image

In preview site:

image

**Comment by severinbeauvais** [comment]
Testing a name-change NR with a C company:

In Dev
image

In preview site
image

**Review by rarmitag — APPROVED** [review]

**Pull request merged**